src/dtgtk/thumbtable.c

```diff
@@ -1793,6 +1793,8 @@ void dt_thumbtable_set_parent(dt_thumbtable_t *table, GtkWidget *new_parent, dt_
       {
         gtk_overlay_reorder_overlay(GTK_OVERLAY(dt_ui_center_base(darktable.gui->ui)),
                                     gtk_widget_get_parent(dt_ui_log_msg(darktable.gui->ui)), -1);
+        gtk_overlay_reorder_overlay(GTK_OVERLAY(dt_ui_center_base(darktable.gui->ui)),
```

Comment:
As for map and lighttable we probably don't need this one.

Reply:
Similarly it's mostly there to future-proof (and because I basically did a straight duplication of the existing log messages) but I'll remove it if you want.